exec-ui/src/app/modules/metrics/modules/metrics/traceability/traceability.configuration.ts

```diff
@@ -0,0 +1,14 @@
+export class TraceabilityConfiguration {
+    public static identifier = 'traceability';
+    public static previewHeading = 'Traceability';
+    public static detailHeading = 'Traceability';
+    public static graphHeading = 'Traceability';
+    public static buildingBlockLabel = 'Traceability';
+    public static description = 'Traceability is the percentage of feature files tagged with Story IDs from test execution results / Total number of feature files found in Git.';
```

Comment:
Is the metric Git specific? Or does it work with other source code management tools?

Reply:
it is not git specific. updated the statement to be generic.